### PR TITLE
tests, utils: Move a few helpers out of `tests.go` famous file.

### PR DESCRIPTION
**Pull request by EdDev**
**What this PR does / why we need it**:

The `utils.go` file is huge with a mix of unrelated helpers.
    
- Move the `CheckResultShellCommandOnVmi` function closer to where it is used.
- Move the pod rendering files to its own package.
    
**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Checklist**

This checklist is not enforcing, but it's a reminder of items that could be relevant to every PR.
Approvers are expected to review this list.

- [ ] Design: A [design document](https://github.com/kubevirt/community/tree/main/design-proposals) was considered and is present (link) or not required
- [ ] PR: The PR description is expressive enough and will help future contributors
- [ ] Code: [Write code that humans can understand](https://en.wikiquote.org/wiki/Martin_Fowler#code-for-humans) and [Keep it simple](https://en.wikipedia.org/wiki/KISS_principle)
- [ ] Refactor: You have [left the code cleaner than you found it (Boy Scout Rule)](https://learning.oreilly.com/library/view/97-things-every/9780596809515/ch08.html)
- [ ] Upgrade: Impact of this change on upgrade flows was considered and addressed if required
- [ ] Testing: New code requires [new unit tests](https://github.com/kubevirt/kubevirt/blob/main/docs/reviewer-guide.md#when-is-a-pr-good-enough). New features and bug fixes require at least on e2e test
- [ ] Documentation: A [user-guide update](https://github.com/kubevirt/user-guide/) was considered and is present (link) or not required. You want a user-guide update if it's a user facing feature / API change.
- [ ] Community: Announcement to [kubevirt-dev](https://groups.google.com/g/kubevirt-dev/) was considered

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: tests/libnet/job/job.go [hunk 1]
```diff
@@ -24,12 +24,13 @@ import (
 	"fmt"
 	"time"
 
+	"kubevirt.io/kubevirt/tests/libpod"
```
Comment (ShellyKa13) [inline tests/libnet/job/job.go:27]: this import should be next to the other kubevirt.io/kubevirt/tests... imports

File: tests/mdev_configuration_allocation_test.go [hunk 2]
```diff
@@ -8,6 +8,8 @@ import (
 	"strings"
 	"time"
 
+	"kubevirt.io/kubevirt/tests/libpod"
```
Comment (ShellyKa13) [inline tests/mdev_configuration_allocation_test.go:11]: same here

**Review by ShellyKa13 — COMMENTED** [review]
Hi overall seems great, small nitpick if you can move the import in all the files instead of a new line next to other "kubevirt.io/kubevirt/tests/..." imports

**Comment by AlonaKaplan** [comment]
/approve

Please fix the import issues before merging.

**Comment by EdDev** [comment]
change: Rebase and re-arranged the imports per the review comments.

File: tests/libnet/job/job.go (hunk 1 above)
Comment (EdDev) [reply tests/libnet/job/job.go]: Done

File: tests/mdev_configuration_allocation_test.go (hunk 2 above)
Comment (EdDev) [reply tests/mdev_configuration_allocation_test.go]: Done

**Review by ShellyKa13 — COMMENTED** [review]
/lgtm

**Review by ShellyKa13 — COMMENTED** [review]
/lgtm

**Pull request merged**